roles/solrcloud/tasks/config.yml

```diff
@@ -1,65 +1,65 @@
 ---
-- name: solrcloud | Check for solr znode
+- name: Solrcloud | Check for solr znode
   ansible.builtin.command: >
     {{ solr_installation }}/bin/solr zk ls /
     -z {{ solr_zookeeper_hosts_string }}
   register: znode_check
   ignore_errors: true
   changed_when: false
 
-- name: solrcloud | Create zk chroot / solr znode using bin/solr
+- name: Solrcloud | Create zk chroot / solr znode using bin/solr
   ansible.builtin.command: >
     {{ solr_installation }}/bin/solr zk mkroot {{ solr_znode }}
     -z {{ solr_zookeeper_hosts_string }}
   when: solr_znode not in znode_check.stdout_lines and solr_znode != ''
   changed_when: false
 
-- name: solrcloud | Configure SolrCloud init script
+- name: Solrcloud | Configure SolrCloud init script
   ansible.builtin.template:
     src: 'solr.in.sh.j2'
     dest: '/etc/default/solr.in.sh'
     owner: '{{ solr_user }}'
     group: '{{ solr_group }}'
-    mode: 0644
+    mode: "0644"
     force: true
   changed_when: false
   notify: restart SolrCloud
 
-- name: solrcloud | Configure high ulimit value for Solr
+- name: Solrcloud | Configure high ulimit value for Solr
   ansible.builtin.template:
     src: 'solr.conf.j2'
     dest: '/etc/security/limits.d/solr.conf'
-    mode: 0644
+    mode: "0644"
     force: true
   notify: restart SolrCloud
 
-- name: solrcloud | Configure SolrCloud properties
+- name: Solrcloud | Configure SolrCloud properties
   ansible.builtin.template:
     src: 'solr.xml.j2'
     dest: '{{ solr_data_dir }}/solr.xml'
     owner: '{{ solr_user }}'
     group: '{{ solr_group }}'
-    mode: 0644
+    mode: "0644"
     force: true
   notify: restart SolrCloud
 
-- name: solrcloud | Configure pre 7.4 logging
+- name: Solrcloud | Configure SolrCloud 9 properties
   ansible.builtin.template:
-    src: 'log4j.properties.j2'
-    dest: '/solr/log4j.properties'
+    src: 'solr9.xml.j2'
```

Comment:
should this only happen if you're installing solr 9? Or does this role only do solr 9 at this point?

Reply:
no we do NOT!